ui/appui-abstract/src/appui-abstract/BaseUiItemsProvider.ts

```diff
@@ -16,9 +16,9 @@ import { StageUsage } from "./items/StageUsage";
 import { UiItemsProvider } from "./UiItemsProvider";
 import { UiItemsManager } from "./UiItemsManager";
 
-/**
- * Base implementation of a UiItemsProvider. The base class allows the user to pass in a function that is used to determine if the
+/** Base implementation of a UiItemsProvider. The base class allows the user to pass in a function that is used to determine if the
  * active stage should be provided items. Derived provider classes should override the `xxxInternal` methods to provide items.
+ * @deprecated in 3.5. Use [BaseUiItemsProvider]($appui-react) instead.
```

Comment:
Unless you intend to backport this PR, your new deprecations will be in 3.6, not 3.5. The minor version doesn't really matter much - I mostly just want to confirm you don't intend to backport?

Reply:
Oh, you are absolutely correct about that. I do not intend to backport this PR.

Reply:
Fixed.